test/unit/mods.example.ts

```diff
@@ -667,13 +670,124 @@ export const JsonModShouldFail = new Map<string, ExampleFailingMod>(
   }),
 );
 
+export const IniMod = new Map<string, ExampleMod>(
+  Object.entries({
+    archiveWithSingleIniAtRoot: {
```

Comment:
Ah! Just use `iniWithSingle[Reshade]IniAtRoot` etc. :) `archive*` is overloaded to `*.archive` mods here

Reply:
done :)